libcloud/__init__.py

```diff
@@ -75,3 +94,40 @@ def _init_once():
             paramiko.common.logging.basicConfig(level=paramiko.common.DEBUG)
 
 _init_once()
+
+
+class DriverType:
+    """ Backup-as-a-service driver """
```

Comment:
I would also prefer to move this code to `base.py` or similar module since usually the convention is for init to either contain initialization code or be empty (and this is not really initialization code).

We can of course also still import DriverType and get_driver from other module here so user can still do "from libcloud import get_driver" for convenience.